internal/pkg/cli/flag.go

```diff
@@ -237,6 +237,7 @@ Defaults to all logs. Only one of end-time / follow may be used.`
 	pipelineResourcesFlagDescription = "Optional. Show the resources in your pipeline."
 	localSvcFlagDescription          = "Only show services in the workspace."
 	localJobFlagDescription          = "Only show jobs in the workspace."
+	localPipelineFlagDescription     = "Only show pipelines in the workspace."
```

Comment:
This is (correctly) consistent with the other `--local` flags, but just a note to ourselves that we should go through all of the flags to make sure we include "Optional" where should.